src/App.jsx

```diff
@@ -14,70 +16,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGoods] = useState(goods[8]);
```

Comment:
```suggestion
  const [selectedGood, setSelectedGoods] = useState(goods[8]);
```
In react there is a convention on how to name your states:
![image](https://github.com/mate-academy/react_goods-selector-js/assets/63243136/dbaec9cb-96fa-42a4-aacf-777b9dafc92c)

https://react.dev/reference/react/useState